### PR TITLE
DSP-377 Support Islamic calendar

### PR DESCRIPTION
**Pull request by SepidehAlassi**
[DSP-377](https://dasch.myjetbrains.com/youtrack/issue/DSP-377)

- [x] Support Islamic calendar
- [x] add tests
- [x] update documentation
should resolve [issue #1591 ](https://github.com/dasch-swiss/knora-api/issues/1591)

### CODE REVIEW

File: webapi/src/test/scala/org/knora/webapi/messages/util/CalendarDateUtilV2Spec.scala [hunk 1]
```diff
@@ -259,5 +261,78 @@ class CalendarDateUtilV2Spec extends CoreSpec() {
                 calendarDateRange.toJulianDayRange
             }
         }
+
+        //*** Test ISLAMIC Date Conversions ****//
+        "convert an islamic date string to a CalendarDateV2 BC" in {
```
Comment (benjamingeer) [inline webapi/src/test/scala/org/knora/webapi/messages/util/CalendarDateUtilV2Spec.scala:266]: Is BC a typo here?

**Review by benjamingeer — CHANGES_REQUESTED** [review]
It's great to see this!

File: webapi/src/test/scala/org/knora/webapi/messages/util/CalendarDateUtilV2Spec.scala (hunk 1 above)
Comment (SepidehAlassi) [reply webapi/src/test/scala/org/knora/webapi/messages/util/CalendarDateUtilV2Spec.scala:266]: oh yes

**Review by benjamingeer — APPROVED** [review]
Looks great, thank you!

**Comment by SepidehAlassi** [comment]
> Looks great, thank you!

Thank you for reviewing this!

**Pull request merged**